the-alexa-skill/python/the_alexa_skill/the_alexa_skill_stack.py

```diff
@@ -46,11 +46,7 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
                                       removal_policy=core.RemovalPolicy.DESTROY
                                       )
 
-        # install node dependencies for lambdas
-        lambda_folder = os.path.dirname(os.path.realpath(__file__)) + "/../lambda_fns"
-        subprocess.check_call("npm i".split(), cwd=lambda_folder, stdout=subprocess.DEVNULL)
-        subprocess.check_call("npm run build".split(), cwd=lambda_folder, stdout=subprocess.DEVNULL)
-
```

Comment:
I know you don't oppinate about stacks, but I think (I just think) that this case is worth considering.
In all other patterns where the runtime is NODEJS_12_X, the function is synthesized from a .ts in the TS version and the .js file is copied to the Python version, so I suppose that this pattern must be kept in the other stacks. In addition, I think it makes the deployment process simpler, without having to compile the .ts every time.